app/views/spotlight/sir_trevor/blocks/_quote_block.html.erb

```diff
@@ -1,7 +1,13 @@
 <div class="st__content-block st__content-block--quote">
   <div class="quote">
     <div class="quote__content">
-    	<%= sir_trevor_markdown quote_block.text %>
+      <% if quote_block.format == :html %>
+        <blockquote>
+          <%= sir_trevor_format quote_block.text, format: quote_block.format %>
+        </blockquote>
+      <% else %>
+        <%= sir_trevor_markdown quote_block.text %>
+      <% end %>
```

Comment:
one option to offset both the quotation and the attribution more clearly would be to wrap the entire `<div class="quote">` in `<blockquote>`.  but @ggeisler says he's also fine w/ the sir trevor default, so if that's lower maintenance burden (i.e. if we're not doing any other customization of our own here), then we could just leave this as-is.  i have no strong opinion on it myself.

see also: https://github.com/projectblacklight/spotlight/pull/1651#issuecomment-267193593